apps/condo/domains/resident/tasks/notifyResidentsOnPaydayTask.js

```diff
@@ -0,0 +1,37 @@
+const dayjs = require('dayjs')
+
+const { featureToggleManager } = require('@open-condo/featureflags/featureToggleManager')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { createCronTask } = require('@open-condo/keystone/tasks')
+
+const { SEND_BILLING_RECEIPTS_ON_PAYDAY_REMAINDER_TASK } = require('@condo/domains/common/constants/featureflags')
+
+const { notifyOnPaydayIsCurrentDateValid } = require('./helpers/notifyOnPaydayisCurrnetDateValid')
+const { notifyResidentsOnPayday } = require('./notifyResidentsOnPayday')
+
+
+const logger = getLogger('meter/sendSubmitMeterReadingsPushNotifications')
+
+const notifyResidentsOnPaydayTaskFn = async (context = null) => {
```

Comment:
When we export result of `createCronTask` without intermediate constant, function name can be simplified.
Btw, this function is not a task, it is a worker.
Task 
```suggestion
const notifyResidentsOnPaydayWorker = async (context = null) => {
```